lambda-functions/notification-es-logging-service/src/main/resources/log4j2.xml

```diff
@@ -7,7 +7,7 @@
     </Appenders>
     <Loggers>
 		<!-- avoid duplicated logs with additivity=false -->
-        <Logger name="com.paladincloud" level="info" additivity="false">
+        <Logger name="com.paladincloud" level="error" additivity="false">
```

Comment:
Should we not use LOG_LEVEL here?

Reply:
Good catch. yes we need to use LOG_LEVEL here.  Fixed it. Probably lost during shelve.